src/compiler/crystal/tools/doc/markdown/doc_renderer.cr

```diff
@@ -30,78 +30,53 @@ class Crystal::Doc::Markdown::DocRenderer < Crystal::Doc::Markdown::HTMLRenderer
   def end_inline_code
     @inside_inline_code = false
 
-    text = @code_buffer.to_s
+    @io << detect_code_link(@code_buffer.to_s)
+    super
+  end
 
+  def detect_code_link(text : String) : String
```

Comment:
I'd suggest to rename this method to `expand_code_links` to more precisely communicate its purpose.

Reply:
Good idea. Done.